presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -248,9 +248,14 @@ public void testParquetUseColumnNames(int rowCount)
     {
         // Write of complex hive data to Parquet is broken
         // TODO: empty arrays or maps with null keys don't seem to work
+        // Parquet does not support DATE
```

Comment:
I had to add the DATE check for completeness. Should I make it a separate commit or is it OK?